ironic-deployment/keepalived-tls/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: metal3
+resources:
+- ../keepalived
+
+configMapGenerator:
+- behavior: merge
+  literals:
+  - CACERT_FILE=/tls/ca/ca.crt
+  - CERT_FILE=/tls/cert/tls.crt
+  - KEY_FILE=/tls/cert/tls.key
+  name: ironic-bmo-configmap
+
+secretGenerator:
+- name: ironic-cacert
+  files:
+  - ca.crt=ca.crt
+- name: ironic-cert
+  files:
+  - tls.crt=tls.crt
+  - tls.key=tls.key
```

Comment:
Only this is actually a secret.

Reply:
Indeed, Ironic itself probably does not need the CA, only BMO does

Reply:
I added the behaviour of using the certificate as CA for ironic, so now we can remove this.